### PR TITLE
Fixed spelling mistakes and comma punctuation errors

### PR DESCRIPTION
**Pull request by wyckster**
I fixed several spelling mistakes and punctuation errors.

I know the use of a comma after a short introductory phrase is considered optional for phrases of four words or less, but Microsoft's grammar checker prefers the comma to be there, so I made the changes.  I hope that's not contentious.

### CODE REVIEW

File: docs/extensions/themes-snippets-colorizers.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The easiest way to create a new workbench color theme is to start with an existi
 
 - Switch to the color theme that you want to modify.
 - Open the [settings](/docs/getstarted/settings.md) and make changes to view and editor colors using the `workbench.colorCustomizations`. Changes are applied live to your VS Code instance and no refreshing or reloading is necessary.
-- A complete list of all themable colors can be found in the [color reference](https://code.visualstudio.com/docs/getstarted/theme-color-reference).
+- A complete list of all themeable colors can be found in the [color reference](https://code.visualstudio.com/docs/getstarted/theme-color-reference).
```
Comment (gregvanl) [inline docs/extensions/themes-snippets-colorizers.md]: This is a tricky one. We've gone with themable in our APIs so we'll stick with themable in our docs as well. I'll revert after I merge the PR.

**Comment by gregvanl** [comment]
Thanks!

File: docs/extensions/themes-snippets-colorizers.md (hunk 1 above)
Comment (wyckster) [reply docs/extensions/themes-snippets-colorizers.md]: Ah!  Understood. Thanks  :)

**Comment by Loridawn99** [comment]
Love you and miss you!

**Pull request merged**